.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I think we should gitignore this since `"python.pythonPath": "/Users/janhenrikbernhard/opt/anaconda3/envs/aml/bin/python",` will interfere with other envs